document/docs/bulkmessage.md

```
@@ -38,4 +38,4 @@ The display name in the LINE column acts as a link. Clicking this link will take
 Check the checkboxes for the target users, select **Send LINE Message** from the bulk action dropdown, and click the **Apply** button.  
 <img src="https://blog.shipweb.jp/wp-content/uploads/2022/01/lineconnect-ss-12.jpg" width="320" />  
 The bulk message page will open with the selected WordPress user IDs pre-specified.  
-<img src="https://blog.shipweb.jp/wp-content/uploads/2022/01/lineconnect-ss-13.jpg" width="320" />  
\ No newline at end of file
+<img src="https://blog.shipweb.jp/wp-content/uploads/2022/01/lineconnect-ss-13.jpg" width="320" />  
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)